sympy/logic/boolalg.py

```diff
@@ -2782,14 +2797,14 @@ def bool_monomial(k, variables):
 
     Each boolean function can be uniquely represented by a
     Zhegalkin Polynomial (Algebraic Normal Form). The Zhegalkin
-    Polynomial of the boolean function with n variables can contain
-    up to 2^n monomials. We can enumarate all the monomials.
+    Polynomial of the boolean function with `n` variables can contain
+    up to `2^n` monomials. We can enumarate all the monomials.
```

Comment:
I also wonder if things like this should use `$2^n$` since the mathdollar sphinx extension is used.

Reply:
I know about that, but I use Spyder and there $ does not work. Backtick doesn't work properly either, but at least turns it into italic and removed the backspaces. Hence, I'm not really keen on the $ as it should do the same thing as backtick (but worse).